pkg/cli/cli.go

```diff
@@ -365,3 +366,35 @@ func printDiskUsage(cfg *config.Config) {
 		}
 	}
 }
+
+func storageCleaner(cfg *config.Config, s *storage.Storage) {
+	logrus.Debug("Storage cleaner scheduled")
+
+	t := time.NewTicker(10 * time.Second)
+	for {
+		<-t.C
+
+		depth := cfg.Server.RetentionThresholdDepth
+		if cfg.Server.RetentionThresholdDepth > cfg.Server.StorageMaxDepth {
+			depth = cfg.Server.StorageMaxDepth
+		}
+
+		labels := []string{
+			"pyroscope.server.cpu",
+			"pyroscope.server.alloc_space",
+			"pyroscope.server.alloc_objects",
+			"pyroscope.server.inuse_space",
+			"pyroscope.server.inuse_objects",
+		}
```

Comment:
I see you hardcoded these for now. We need this list to be generated dynamically and it should include all of the keys out there.

To get a list of all possible keys you can do this:
* call `labels.GetValues` with `__name__` as the key
* for each value, call `dimensions.Get(k+":"+v)` with key being `__name__`, and `v` being a value from GetValues ([example](https://github.com/pyroscope-io/pyroscope/blob/main/pkg/storage/storage.go#L266))